tests/mobly_unittest_suite.py

```diff
@@ -17,26 +17,25 @@
 import sys
 import unittest
 
-import mobly_android_device_test
-import mobly_asserts_test
-import mobly_base_class_test
-import mobly_logger_test
-import mobly_records_test
-import mobly_sl4a_client_test
-import mobly_test_runner_test
-import mobly_utils_test
-
+from tests.mobly import asserts_test
```

Comment:
i think this file is no longer needed since the tests are run by setup.py's 'test' action now.